test/session.go

```diff
@@ -163,7 +214,7 @@ var sessionAssets = `[
 ]`
 
 var sessionTrigger = `{
-    "type": "manual",
+    "type": "flow_action",
```

Comment:
changed this to a flow_action trigger as an easy of having `@parent` set in tests